client/allocrunner/taskrunner/sids_hook.go

```diff
@@ -51,6 +51,7 @@ type sidsHookConfig struct {
 	sidsClient consul.ServiceIdentityAPI
 	lifecycle  ti.TaskLifecycle
 	logger     hclog.Logger
+	runner     *TaskRunner
```

Comment:
I think we could just use `hookResources *cstructs.AllocHookResources` like the `consul_hook` does? There's no need to have the full `TaskRunner` object I think?

Reply:
Good call, will fix.